js/src/framework.ts

```diff
@@ -383,10 +385,18 @@ export async function runEvaluator(
               const result = await rootSpan.traced(
                 async (span: Span) => {
                   const scoreResult = score(scoringArgs);
-                  const result =
+                  const scoreValue =
                     scoreResult instanceof Promise
                       ? await scoreResult
                       : scoreResult;
+
+                  if (scoreValue === null) {
+                    return null;
+                  }
+                  const result: Score =
+                    scoreValue === null || typeof scoreValue === "object"
```

Comment:
We're checking for null above, so the first case should be unnecessary? Maybe just `typeof scoreValue === 'number'`?